src/ml_board/README.md

```diff
@@ -70,6 +70,31 @@ payload:
 }
 ```
 
+**Model Card**
+
+Fetch System details like CPU & GPU for model card.
+
+_GET /system-info_
```

Comment:
I think we should have this information for each experiment in a  gridsearch. We could add it to the logging files like `<gridsearch_id>/<experiment_id>/environment.json`. 

You are querying this information on the fly for each request. Note that the API server might be running not on the same node as the MLgym computation part.

Reply:
Additional information would be helplful as well such as python version and the installed python packages.

Reply:
> I think we should have this information for each experiment in a gridsearch. We could add it to the logging files like `<gridsearch_id>/<experiment_id>/environment.json`.
> 
> You are querying this information on the fly for each request. Note that the API server might be running not on the same node as the MLgym computation part.

this is done